src/Debug.js

```diff
@@ -63,3 +73,61 @@ export function debugNode(node, indentLength = 2) {
 export function debugNodes(nodes) {
   return nodes.map(debugNode).join('\n\n\n');
 }
+
+export function debugInst(inst, indentLength = 2) {
+  if (typeof inst === 'string' || typeof inst === 'number') return escape(inst);
+  if (!inst) return '';
```

Comment:
In that case, wouldn't we want `if (inst === true) return ''`? or am i misunderstanding something.

what about `<div>{Symbol('foo')}</div>`?